app/Controllers/Items.php

```
@@ -617,8 +617,27 @@ public function getBulkEdit(): string
+                ],
+            ];
+
+            if (!$this->validate($rules, $messages)) {
+                $errors = $this->validator->getErrors();
+
+                return $this->response->setJSON(['success' => false, 'message' => reset($errors), 'id' => $item_id]);
+            }
+        }
+
+        $upload_data = $this->upload_image();
+        $upload_success = empty($upload_data['error']);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$upload_success'. (undefined)

(UnusedLocalVariable)